configs/sandpack-contents/component-theming/alert.js

```diff
@@ -0,0 +1,119 @@
+module.exports = {
+  App: `import {
+  Box,
+  useColorMode,
+  IconButton,
+  Alert,
+  AlertDescription,
+  AlertIcon,
+  AlertTitle,
+} from "@chakra-ui/react";
+import { FaMoon, FaSun } from "react-icons/fa";
+
+export default function App() {
+  const { toggleColorMode, colorMode } = useColorMode();
+
+  return (
+    <Box pt={12} position="relative" h="100vh">
+      <Alert size="md" status="error">
+        <AlertIcon />
+        <AlertTitle>Your browser is outdated!</AlertTitle>
+        <AlertDescription>
+          Your Chakra experience may be degraded.
+        </AlertDescription>
+      </Alert>
+
+      <IconButton
+        aria-label="toggle theme"
+        rounded="full"
+        size="xs"
+        position="absolute"
+        bottom={4}
+        left={4}
+        onClick={toggleColorMode}
+        icon={colorMode === "dark" ? <FaSun /> : <FaMoon />}
+      />
+    </Box>
+  );
+}
+`,
+  Index: `import * as React from "react";
+import { createRoot } from "react-dom/client";
+import { ChakraProvider, extendTheme } from "@chakra-ui/react";
+
+import App from "./App";
+import { alertTheme } from "./theme/components/Alert";
+
+const theme = extendTheme({
+  components: {
+    Alert: alertTheme,
+  }
+});
+
+const container = document.getElementById("root");
+const root = createRoot(container!);
+root.render(
+  <ChakraProvider theme={theme}>
+    <App />
+  </ChakraProvider>
+);`,
+  AlertTheme: `import { alertAnatomy as parts } from "@chakra-ui/anatomy";
+import {
+  createMultiStyleConfigHelpers,
+  defineStyle,
+} from "@chakra-ui/styled-system";
+import { extendTheme } from "@chakra-ui/react";
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(parts.keys);
+
+const baseStyle = definePartsStyle({
+  // define the part you're going to style
+  title: {
+    color: "teal.500", // change the color of the title text of the alert
+  },
+});
+
+// Defining a custom variant called mono
+const mono = definePartsStyle((props) => {
+  const { colorScheme: c } = props;
```

Comment:
You can obtain the `status` from the props here, and color the alert based on it. Try doing `console.log(props)` to see it. I wouldn't use a `colorScheme` in this case. The `status` should control the color of the alert.